docs/content/node/reference/frameworks/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Frameworks
+weight: 3
+any: false
+frameworksTOC: true
```

Comment:
Specifying frameworks to request in the front matter here would be more clear than in config.toml.